src/app/index.tsx

```diff
@@ -13,7 +13,17 @@ const cols = _range(0, 100).map((_idx) => ({} as any));
 rows[0].header = true;
 rows[0].height = 50;
 
+const renderer = (row: number, col: number, data: { formatted: string }) => {
+  // if (col % 2) {
+  //   return <b>{data.formatted}</b>;
+  // }
+  if (col === 0) {
+    return <a>{data.formatted}</a>;
```

Comment:
This is just for demo purposes, right? I think this needs an `href` to look like a link.

Reply:
correct just for demo. it still looks "linky" even tho it doesn't do anything haha

Reply:
btw if you want to try the demo, just `npm run dev` and open `localhost:7998` in your fork